### PR TITLE
Test for #1336

### PR DESCRIPTION
**Pull request by Jurigag**
This is test for https://github.com/phalcon/zephir/issues/1336

### CODE REVIEW

**Comment by Jurigag** [comment]
This issue as i remember mostly happens because of code generation, each catch block is just generated as if when they should be imho `if elseif` @sjinks

File: test/exceptions.zep [hunk 1]
```diff
@@ -80,4 +83,27 @@ class Exceptions
             throw e;
         }
     }
+
+    public function testMultiException(var returnValue, var exception) {
```
Comment (sergeyklay) [inline test/exceptions.zep:87]: Could you please follow common code style

Comment (Jurigag) [reply test/exceptions.zep:87]: Yea sure, will fix it. You mean make it `string` and `<\Exception>` type and intention above?

Comment (sergeyklay) [reply test/exceptions.zep:87]: Yes, and fix braces

**Comment by sjinks** [comment]
Let's move to #1450

**Pull request closed without merging**